cmake/modules/packages/FindDB2.cmake

```diff
@@ -46,7 +46,7 @@ if(UNIX)
         endforeach(db2_libdir)
     endforeach(db2_install_path)
 elseif(WIN32)
-    if (CMAKE_CL_64) # 64-bit build, DB2 64-bit installed
+    if (CMAKE_SIZEOF_VOID_P EQUAL "8") # 64-bit build, DB2 64-bit installed
```

Comment:
can you delete this file entirely? DB2 has been dropped long time ago, so it is totally unused